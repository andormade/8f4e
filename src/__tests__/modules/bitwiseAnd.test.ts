import moduleTypeGenerator from '../../modules/bitwiseAnd';

it('generates correct module type descriptor', () => {
	expect(moduleTypeGenerator({ vGrid: 6, hGrid: 14 })).toMatchSnapshot();
});
