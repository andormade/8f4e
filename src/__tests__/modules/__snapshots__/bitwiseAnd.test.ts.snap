// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates correct module type descriptor 1`] = `
Object {
  "buttons": Array [],
  "category": "Bitwise",
  "engine": Object {
    "config": Object {},
    "name": "bitwiseAnd",
  },
  "height": 126,
  "initialState": Object {},
  "inputs": Array [
    Object {
      "height": 13,
      "id": "in:1",
      "label": "in",
      "width": 12,
      "x": 12,
      "y": 42,
    },
    Object {
      "height": 13,
      "id": "in:2",
      "label": "in",
      "width": 12,
      "x": 12,
      "y": 56,
    },
  ],
  "lines": Array [
    Object {
      "height": 1,
      "width": 96,
      "x": 6,
      "y": 7,
    },
    Object {
      "height": 112,
      "width": 1,
      "x": 102,
      "y": 7,
    },
    Object {
      "height": 1,
      "width": 96,
      "x": 6,
      "y": 119,
    },
    Object {
      "height": 112,
      "width": 1,
      "x": 6,
      "y": 7,
    },
  ],
  "name": "Bitwise AND",
  "outputs": Array [
    Object {
      "height": 13,
      "id": "out",
      "width": 12,
      "x": 84,
      "y": 42,
    },
  ],
  "sliders": Array [],
  "steppers": Array [],
  "width": 108,
}
`;
